helm-chart/ray-cluster/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for ray-cluster.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: rayproject/ray
+  tag: latest
+  pullPolicy: IfNotPresent
+
+nameOverride: "ray"
+fullnameOverride: ""
+
+head:
+  groupName: headgroup
+  replicas: 1
+  type: head
+  labels:
+    key: value
+  initArgs:
+    port: '6379'
+    object-manager-port: '12345'
+    node-manager-port: '12346'
+    object-store-memory: '100000000'
+    redis-password: 'LetMeInRay'
```

Comment:
default to not having a password?

Reply:
I refer to the file in `config/samples/`. redis-password is necessary. Otherwise, the pod will report a `redis.exceptions.ResponseError: WRONGPASS invalid username-password pair`.